hpx/runtime/threads/detail/thread_pool_base.hpp

```diff
@@ -80,7 +80,8 @@ namespace hpx { namespace threads { namespace detail
         virtual void stop(
             std::unique_lock<compat::mutex>& l, bool blocking = true) = 0;
 
-        virtual void resume(error_code& ec = throws) = 0;
+        virtual void suspend(error_code& ec = throws) {};
+        virtual void resume(error_code& ec = throws) {};
```

Comment:
Should the default implementations at least assert, if not throw an exception instead of silently ignoring the request?

Reply:
Thanks for this fix. Looks fine now.